packages/components/src/components/SquiggleItem.tsx

```diff
@@ -0,0 +1,250 @@
+import * as React from "react";
```

Comment:
Can you come up with a name more specific than ``SquiggleItem``? Maybe ``SquiggleTypedResult`` or something?

Reply:
`SquiggleExpressionViewer`?